### PR TITLE
Added Mayhem Support

### PR DESCRIPTION
**Pull request by matl4c**

### CODE REVIEW

File: mayhem/Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+project: stegano
```
Comment (xansec) [inline mayhem/Mayhemfile:1]: Please omit image name here. Any reason for the 8 second timeout?

File: mayhem/fuzz-stegano.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/python3
```
Comment (xansec) [inline mayhem/fuzz-stegano.py:1]: Are these valid exceptions or simply fuzzing errors?

**Review by xansec — CHANGES_REQUESTED** [review]
Please don't delete existing workflows. Please make the requested changes.

File: mayhem/Mayhemfile (hunk 1 above)
Comment (matl4c) [reply mayhem/Mayhemfile:1]: The timeout is not needed, I will remove it. Should I remove image name on all of my PRs?

File: mayhem/fuzz-stegano.py (hunk 2 above)
Comment (matl4c) [reply mayhem/fuzz-stegano.py:1]: These are either expected exceptions, or exceptions in underlying libraries most notably PIL.

**Review by xansec — CHANGES_REQUESTED** [review]
Yes, please remove the image name from the Mayhemfile. Can you please also restore the deleted workflow? Please make the requested changes.

**Comment by matl4c** [comment]
corrections completed

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**